wordle-web/src/components/KeyBoard.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <v-row class="justify-center" dense v-for="(key, i) in keyboardLetters" :key="i">
+    <v-col cols="auto" v-for="(letter, j) in key" :key="j">
+      <LetterButton 
+      size="small"
+      height="75"
+      min-width="40"
+      :letter="letter"
+       @click="letterClick(letter)" />
```

Comment:
```suggestion
      <LetterButton 
      size="small"
      height="75"
      min-width="40"
      :letter="letter"
      elevation="7"
       @click="letterClick(letter)" />
```
This will give you the drop shadow effect on your keyboard keys